receiver/awsfirehosereceiver/internal/unmarshaler/firehoselog/firehoselog.go

```diff
@@ -0,0 +1,10 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package firehoselog // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsfirehosereceiver/internal/unmarshaler/firehoselog"
+
+type firehoseLog struct {
+	Timestamp   int64  `json:"timestamp"`
+	FirehoseARN string `json:"firehoseARN"`
+	Message     string `json:"message"`
+}
```

Comment:
Nothing produces JSON with these fields does it? IIUC it's just for passing around in memory? If that's the case, I'd suggest dropping the `json:"..."` tags, as they confused me, and may confuse others.

This is unlike cwlog, where the `cWLog` struct holds the JSON structure of logs coming from CloudWatch.